app/script/view_model/content/CollectionDetailsViewModel.coffee

```diff
@@ -39,9 +41,17 @@ class z.ViewModel.content.CollectionDetailsViewModel
     @conversation_et null
     @items.removeAll()
 
+  should_show_header: (message_et) =>
+    if not @last_message_timestamp? or moment(message_et.timestamp).format('M') isnt moment(@last_message_timestamp).format('M')
```

Comment:
No the thing is, i dont like skipping the brackets in expressions since its quite error prone